From 80c0eca0830dc3fbdb3a6460a395ffa22ad761cb Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Mon, 30 Nov 2020 22:50:50 +0800
Subject: [PATCH 4/6] arm64: rockchip: dts: rk3399: disable cdn_dp on Pinebook
 Pro

The corresponding extcon support for fusb302 is not yet available, and
thus the Type-C controller is not tied with the DP controller. However
the DP controller do not support such configuration, and will fail to
probe, and prevent the full display pipeline from being probed.

Disable cdn_dp controller now for Pinebook Pro, in order to make at
least internal eDP panel work.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 219b7507a10f..4297c1db5a41 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -379,10 +379,6 @@ mains_charger: dc-charger {
 	};
 };
 
-&cdn_dp {
-	status = "okay";
-};
-
 &cpu_b0 {
 	cpu-supply = <&vdd_cpu_b>;
 };
-- 
2.30.0

